SUPPORTED-FORMATS.md

```diff
@@ -2178,6 +2178,27 @@ analyze - iccxxxxcompiler_opts cstat2.c</pre></code>For details check the IAR C-
                 -
             </td>
         </tr>
+        <tr>
+            <td>
+                vale
+            </td>
+            <td>
+                -
+            </td>
+            <td>
+                <a href="https://vale.sh">
+                    Vale
+                </a>
+            </td>
+            <td>
+                **/vale-report.json
+            </td>
+        </tr>
+        <tr>
+            <td colspan="4">
+                :bulb: Use option --output=JSON
+            </td>
+        </tr>
```

Comment:
Revert this part as this file is generated (on a merge to main).

Reply:
Done